impeller/display_list/display_list_dispatcher.cc

```diff
@@ -371,11 +379,9 @@ static std::optional<Rect> ToRect(const SkRect* rect) {
 void DisplayListDispatcher::saveLayer(const SkRect* bounds,
                                       const flutter::SaveLayerOptions options,
                                       const flutter::DlImageFilter* backdrop) {
-  if (backdrop) {
-    UNIMPLEMENTED;
-  }
-  canvas_.SaveLayer(options.renders_with_attributes() ? paint_ : Paint{},
-                    ToRect(bounds));
+  auto paint = options.renders_with_attributes() ? paint_ : Paint{};
+  canvas_.SaveLayer(paint, ToRect(bounds),
+                    backdrop ? ToImageFilterProc(backdrop) : std::nullopt);
```

Comment:
I was thinking more along the lines of just:

```
canvas_.SaveLayer(paint, ToRect(bounds), ToImageFilterProc(backdrop));
```

since ToIFP handles null already...

Reply:
Whoopsies, went too fast here. Fixed.